src/components/Input/TextField/TextFieldUtils.ts

```diff
@@ -20,3 +24,17 @@ export function getProperTextFieldBgColor({
   if (variant === TextFieldVariant.Primary && (focused || hasError)) { return 'bg-white-normal' }
   return 'bg-grey-lightest'
 }
+
+interface BorderRadiusProps {
+  size: TextFieldSize
+}
+
+export function getProperTextFieldBorderRadius({
+  size,
+}: BorderRadiusProps): FlattenSimpleInterpolation {
+  if (size > TextFieldSize.L) {
+    return Rounding.round12
+  }
```

Comment:
size를 마치 enum처럼 다뤄주시면 좋을 것 같습니다.

```suggestion
  if (size === TextFieldSize.XL) {
    return Rounding.round12
  }
```

(또는 switch-case)